tools/internal_ci/linux/grpc_interop_tocloud.sh

```diff
@@ -36,5 +36,6 @@ export LANG=en_US.UTF-8
 cd $(dirname $0)/../../..
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
+source tools/internal_ci/helper_scripts/prepare_build_interop_rc
 
 tools/run_tests/run_interop_tests.py -l all -s all --use_docker --http2_interop -t -j 12 $@
```

Comment:
In order to get interop tests building in a reasonable amount of time, you'll probably need to implement the DOCKERHUB_ORGANIZATION trick here:
https://github.com/grpc/grpc/blob/master/tools/run_tests/dockerize/build_interop_image.sh#L83

(and upload the interop base images to dockerhub).